### ISSUE TITLE
Results on YouCook2 varies with different Runs - Seed is same!

### ISSUE THREAD

**Issue by anilbatra2185** [ttengwang/PDVC#3]
Hi @ttengwang !

thanks for nice work on DVC!

I am able to run the code on YouCook2 with small configuration changes, however, I am getter different results if run the same code multiple time with same seed. Metric is **SODA_c**.

| Method  | Validation    |
| ------- | ------------- |
| Run 1   | 4.171         |
| Run 2   | 3.933         |
| Run 3   | 4.322         |
| Run 4   | 3.958         |
| Average | 4.096 ± 0.159 |

Please let me know your thoughts!

Regards
Anil

**Comment by ttengwang** [comment]
Hi, sorry for the late reply! I also noticed that the training process has randomness, but I am not very sure about this question. A possible reason is the non-determinism in [torch.nn.LSTM](https://pytorch.org/docs/stable/generated/torch.nn.LSTM.html#torch.nn.LSTM).
